### ISSUE TITLE
'Latest Changes' does not report changes to directories

### ISSUE THREAD

**Issue by uglygus** [syncthing/syncthing#3896]
'Latest Change' only reports changes for files not directories.

Create, delete or rename a directory inside a share on one machine...

**What I expect:** 
Latest changes on the other machine to be updated with the name of the directory and 'deleted' or 'updated' 

**What happens** 
The change is propagated and the 'Global Changes' log is updated but the 'Latest Change' does not change.

### Version Information

Syncthing Version: v0.14.19
OS Version: OSX Sierra
Browser Version:Chrome Version 55.0.2883.95 (64-bit)

**Comment by nrm21** [comment]
Do we even still need this feature?  It was added before we had a global log but now that one exists... do we even need it?  Just curious.

**Comment by scienmind** [comment]
> Do we even still need this feature?

I kinda like it - gives you a quick look at repo's state, so I'd say we do :wink:. 
Global log don't link changes to repositories.
Sometimes you are interested in seeing what was last for repository X - and:
1 - if 100000 files updated in other repo's you won't see it in Global Log.
2 - if you have some weird setup, it may not be obvious to infer repository from the file path in global log

**Comment by canton7** [comment]
Yeah, the "Latest change" doesn't get cleared when you restart Syncthing, and is per-folder.
